tests/conftest.py

```diff
@@ -81,6 +82,11 @@ def java_command_line(tmp_path: Path, java_args: List[str]) -> List[str]:
 
 @fixture
 def command_line(runtime: str, java_command_line: List[str]) -> List[str]:
+    # these do not have non-container application - so it will result in an error if the command
+    # line is used.
+    if runtime.startswith("native"):
+        return ["/bin/false"]
+
```

Comment:
Looks like you are just piggybacking the `application`s family of fixture whilst you actually want a fixture of your own..

Reply:
You're right. I plan to address that later when I add tests for py-spy and PyPerf across the full matrix of (musl, python_version)